lib/Tunnel.js

```diff
@@ -59,12 +69,21 @@ Tunnel.prototype._init = function(cb) {
                 max_conn: max_conn
             });
         })
-        .catch(function(err){
-            // TODO (shtylman) don't print to stdout?
-            console.log('tunnel server offline: ' + err.message + ', retry 1s');
-            return setTimeout(get_url, 1000);
+        .catch(function(attemptErr){
+            if (op.retry(attemptErr)) {
+                // TODO (shtylman) don't print to stderr?
```

Comment:
There's still a couple todos in the PR

Reply:
Whitespace changes aside, this PR merely changes the comment's text (I will revert it though, as it currently *does* log to `stdout`), but it neither introduces the problem/task nor does it fix it.